import_video.py

```diff
@@ -66,8 +63,9 @@ def get_params(self, path):
                     lng = float(match.group('lng'))
                 except: pass
         if not camera:
-            logging.warn('did not detect camera name, assuming "unnamed"')
-            camera = DEFAULT_CAMERA_NAME
+            logging.error('did not detect camera name, assuming "%s"', self.DEFAULT_CAMERA_NAME)
+            sys.exit(1)
+            camera = self.DEFAULT_CAMERA_NAME
```

Comment:
it is logged on line 53 when we match the regex, this if statement catches the case where we didn't match the regex and couldn't find a name